<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadarDomSimulationWidget</class>
 <widget class="QWidget" name="RadarDomSimulationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>733</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_position">
     <property name="title">
      <string>Position Orientation</string>
     </property>
     <layout class="QFormLayout" name="formLayout_pos">
      <item row="1" column="0">
       <widget class="QLabel" name="label_lon">
        <property name="text">
         <string>Longitude (°):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_lon">
        <property name="text">
         <string>75.000000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_lat">
        <property name="text">
         <string>Latitude (°):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_lat">
        <property name="text">
         <string>22.000000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_alt">
        <property name="text">
         <string>Altitude (m):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_alt">
        <property name="text">
         <string>100.0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_az">
        <property name="text">
         <string>Azimuth (°):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_az">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_el">
        <property name="text">
         <string>Elevation (°):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_el">
        <property name="text">
         <string>15.0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_select">
        <property name="text">
         <string>Select Location</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_radar">
     <property name="title">
      <string>Radar Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout_radar">
      <item row="0" column="0">
       <widget class="QLabel" name="label_freq">
        <property name="text">
         <string>Frequency (GHz):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_freq">
        <property name="text">
         <string>9.4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_range">
        <property name="text">
         <string>Range (km):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_range">
        <property name="text">
         <string>50.0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_bw">
        <property name="text">
         <string>Beam Width (°):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_bw">
        <property name="text">
         <string>2.0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_rspeed">
        <property name="text">
         <string>Rotation Speed (°/s):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_rspeed">
        <property name="text">
         <string>15.0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_power">
        <property name="text">
         <string>Power (kW):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_power">
        <property name="text">
         <string>100.0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_gain">
        <property name="text">
         <string>Antenna Gain (dB):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_gain">
        <property name="text">
         <string>30.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_controls">
     <property name="title">
      <string>Simulation Controls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_update">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_clear">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color:rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>CLOSE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
